main.go

```diff
@@ -52,12 +54,12 @@ func run() int {
 	exportErr := xcodeTestRunner.Export(res, runErr != nil)
 
 	if runErr != nil {
-		logger.Errorf("Run: %s", runErr)
+		logger.Errorf(errorutil.FormattedError(fmt.Errorf("Failed to execute Step main logic: %w", runErr)))
```

Comment:
`Failed to execute Step` is the latest version, based on the [doc](https://bitrise.atlassian.net/wiki/spaces/RD/pages/2445082990/Human+readable+error+messages+for+Steps#3%2C-Fixed-structure-for-first-part-of-error-messages).